SNIClient.py

```diff
@@ -649,7 +652,12 @@ async def game_watcher(ctx: SNIContext) -> None:
 
         perf_counter = time.perf_counter()
 
-        await ctx.client_handler.game_watcher(ctx)
+        try:
+            await ctx.client_handler.game_watcher(ctx)
+        except Exception as ex:
+            ctx.gui_error("Game Exception", ex)
+            snes_logger.exception(ex)
```

Comment:
Doesn't this put the full stack trace in the gui log? I was thinking we would want a shorter message in the gui log, and the full stack trace in just the text file log.